diskqueue_test.go

```diff
@@ -250,32 +250,41 @@ func TestDiskQueueCorruption(t *testing.T) {
 }
 
 type md struct {
-	depth        int64
-	readFileNum  int64
-	writeFileNum int64
-	readPos      int64
-	writePos     int64
+	depth         int64
+	readFileNum   int64
+	writeFileNum  int64
+	readMessages  int64
+	writeMessages int64
+	readPos       int64
+	writePos      int64
 }
 
-func readMetaDataFile(fileName string, retried int) md {
+func readMetaDataFile(fileName string, retried int, withDiskSpaceFeat bool) md {
```

Comment:
nit: in the main file this flag is called `diskLimitFeatIsOn`, we could use the same name to keep consistency.

Reply:
Good call!